project/Dependencies.scala

```diff
@@ -78,13 +78,17 @@ object Dependencies {
   val okhttp = "com.squareup.okhttp3" % "okhttp" % "3.10.0"
   val jsonSchema = "com.eclipsesource"  %% "play-json-schema-validator" % "0.9.5-M4"
 
-  // Fixing transient dependency issue
-  // AWS SDK (1.11.181), which kinesis-logback-appender depends on, brings com.fasterxml.jackson.core and com.fasterxml.jackson.dataformat libs in version 2.6.9
-  // play-json comes with com.fasterxml.core 2.8.9 (without jackson.dataformat) which will evict jackson.core 2.6.9
-  // This lead to incompatible version of jackson.core and jackson.dataformat to be present in the class path
-  // This forces jackson.dataformat to the same version as the one brough by play-json
-  // This line could be remove as soon as the AWS SDK is updated to use the same version coming with play-json
-  val jacksonDataFormat = "com.fasterxml.jackson.dataformat" % "jackson-dataformat-cbor" % "2.8.9"
+  // sbt-native-packager does not seem respect the latestRevision conflict manager when building the
+  // classpath in the executable shell file for the service. The classpath output is different to the
+  // dependencies indicated by the dependency tree plugin. Specifying jackson versions manually seems
+  // to be the only way of making sbt-native-packager build a classpath with consistent jackson versions.
+  val jacksonVersion = "2.11.0"
+  val jacksonDataFormat = "com.fasterxml.jackson.dataformat" % "jackson-dataformat-cbor"  % jacksonVersion
+  val jacksonCore = "com.fasterxml.jackson.core" % "jackson-core" % jacksonVersion
+  val jacksonDataType = "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % jacksonVersion
+  val jacksonDataTypeJdk8 = "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8" % jacksonVersion
+  val jacksonAnnotations = "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonVersion
+  val jackson = Seq(jacksonDataFormat, jacksonCore, jacksonDataType, jacksonAnnotations)
```

Comment:
This solves the classNotFound exceptions from the previous failed attempt to deploy